### PR TITLE
Bullet point in "responsibilities" was missing a direct object.

### PR DESCRIPTION
**Pull request by mikesamuel**
I also fixed a few cosmetic markdown issues and linkified "CTC."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +43,13 @@ The Node.js Security Working Group is _not_ responsible for managing incoming
 security reports to the security@nodejs.org address, nor is it privy to or
 responsible for preparing embargoed security patches and releases.
 
-The Node.js CTC maintains primary responsibility for the management of private
+The [Node.js CTC][] maintains primary responsibility for the management of private
 security activities for Node.js core but relies on the Node.js Security Working
 Group to recommend and help maintain policies and procedures for that
 management.
 
+[Node.js CTC]: https://github.com/nodejs/CTC
```
Comment (cjihrig) [inline README.md]: While you're here, can you change CTC to TSC. The CTC no longer exists.

Comment (cjihrig) [reply README.md]: Oh, and can you move the link table to the bottom of the file.

**Comment by mikesamuel** [comment]
I replaced CTC with TSC.  It was mentioned one place that had not previously been modified.

**Comment by mikesamuel** [comment]
I don't have merge privileges.  Nor do I have privs to assign this to someone who does.  Who can move this forward?

**Comment by mhdawson** [comment]
Out of office for next week, will look at landing when I get back unless somebody beats me to it.

**Pull request merged**